content/api/commands/clearcookies.md

```diff
@@ -2,7 +2,7 @@
 title: clearCookies
 ---
 
-Clear all browser cookies for current domain and subdomain.
+Clear browser cookies for current domain and subdomain.
```

Comment:
didn't the typescript types say something like for the given domain or hostname?